windows/install_gcloud_sdk.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue'
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+Write-Host -ForegroundColor Green Installing Google Cloud SDK
+$gsdk = 'https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-315.0.0-windows-x86_64.zip'
```

Comment:
nit: I would put the SDK version into its own variable, if you think it will change frequently maybe even pass it as an argument so you can modify it easily at build time.